BENCHMARKING.md

```diff
@@ -56,9 +57,9 @@ See `dnsperf -h` for an explanation of the flags.
 
 ### Latest Benchmark
 
-The latest benchmark was run on 2024/10/06 by @whatyouhide, on an 2021 Apple MacBook Pro, 14-inch, M1 Pro CPU, 16 GB of memory.
+The latest benchmark was run on 2024/10/06 by @nelsonvides, on an 2023 Apple MacBook Pro, 16-inch, M3 Pro CPU, 36GB of memory.
```

Comment:
@NelsonVides  I guess it yesterday? today? :)

Reply:
Ups, didn't update the date, it's from a couple of weeks ago, I was just updating docs here. Will do and I'll probably run it again too! 😄

Reply:
Updated. They're actually even better after all the latest changes 😄